test/runnable/ctorpowtests.d

```diff
@@ -96,8 +96,8 @@ static assert( (-4) ^^ -1.0 == 1.0 / -4);
 static assert( 9 ^^ -1.0 == 1.0 / 9);
 
 // Other integers raised to negative powers create an error
-static assert( !is(typeof(2 ^^ -5)));
-static assert( !is(typeof((-2) ^^ -4)));
+static assert( !is(typeof((){ static assert(2 ^^ -5); })));
+static assert( !is(typeof((){ static assert((-2) ^^ -4); })));
```

Comment:
`static assert(!__traits(compiles,...))`

Reply:
I re-added back in the compiler error for negative integral powers, so this test change is no longer required.